src/scripts.js

```diff
@@ -56,23 +60,85 @@ const strideLength = document.getElementById('strideLength');
 const friend1 = document.getElementById('friend1');
 const friend2 = document.getElementById('friend2');
 const friend3 = document.getElementById('friend3');
+const dailyWater = document.getElementById('dailyWater');
+const weeklyWater = document.getElementById('weeklyWater');
+const sleepReport = document.getElementById('sleepReport');
+const weeklySleep = document.getElementById('weeklySleep');
+const dailyActivity = document.getElementById('dailyActivity');
+const weeklyActivity = document.getElementById('weeklyActivity');
+const extraBox = document.getElementById('extraBox');
 
 // event listeners 👇
 window.addEventListener('load', loadUserData);
 
 // functions: handlers and helpers 👇
+//function that randomly selects currentUser
+
 function loadUserData() {
   getAllData().then((data) => {
     //this is retrieving the array of data array from Promise.all in apiCalls.js
-    getCurrentUser(data[0].userData);
+    const userID = getRandomUser(data[0].userData);
+
+    getCurrentUser(data[0].userData, userID);
+    instantiateHydration(data[1].hydrationData, userID);
+    instantiateSleep(data[2].sleepData, userID);
+    console.log(sleepStats);
     displayProfileBox(); //DOM
+    displayDailyHydration('2020/01/22');
```

Comment:
Do we want to think about adding box to the end of the names for all display functions or taking it off of displayProfileBox @rachelJensen?